data/phones/lib/generate_phones_summary.py

```diff
@@ -5,7 +5,8 @@
 import logging
 import os
 
-from typing import Any, Dict, List
+from operator import itemgetter
```

Comment:
Sort of a departure from the normal style here.

Reply:
Good point!